ext/standard/tests/serialize/max_depth.phpt

```diff
@@ -104,8 +104,8 @@ var_dump(is_array(unserialize(
 ?>
 --EXPECTF--
 Invalid max_depth:
-max_depth should be int
-max_depth cannot be negative
+unserialize(): Argument #2 ($options) must contain the 'max_depth' element with type of int, string given
```

Comment:
ok, I know, this is an insane wording, so please suggest me a better.

Reply:
Yeah, this makes it sound like `max_depth` is a required key. I don't think this fits into the argument error messsage format, so I would stick with `unserialize(): 'max_depth'  option must be of type int, string given` or so.

Reply:
Brilliant idea! I like it very much